pyaedt/icepak.py

```diff
@@ -3964,30 +3914,26 @@ def assign_source(
         thermal_condition : str
             Thermal condition. Accepted values are ``"Total Power"``, ``"Surface Heat"``,
             ``"Temperature"``.
-        assignment_value : str or dict
-            Value and units of the input power, surface heat or temperature (depending on
-            ``thermal_condition``). A dictionary can be used for temperature dependent or transient
-             assignment. The dictionary should contain three keys: ``"Type"``, ``"Function"``, and
-             ``"Values"``. Accepted ``"Type"`` values are: ``"Temp Dep"`` and ``"Transient"``.
-             - Accepted values for the ``"Function"`` key are: ``"Linear"``, ``"Power Law"``, ``"Exponential"``,
-             ``"Sinusoidal"``, ``"Square Wave"`` and ``"Piecewise Linear"``. ``"Temp Dep"`` only
-             support the latter. ``"Values"`` contains a list of strings containing the parameters
-            required by the ``"Function"`` selection (e.g. ``"Linear"`` requires two parameters:
-            the value of the variable at t=0 and the slope of the line). The parameters required by
-            each ``Function`` option is in Icepak documentation. The parameters must contain the
-            units where needed.
+        assignment_value : str or dict or BoundaryDictionary
+            Value and units of the input power, surface heat, or temperature (depending on
+            ``thermal_condition``).
+            Assign a transient condition using the result of a function with
+            the pattern ``create_*_transient_assignment``.
```

Comment:
```suggestion
            the ``create_*_transient_assignment`` pattern.
```
Place the noun after the code entity.